oe_theme.theme

```diff
@@ -262,6 +262,59 @@ function oe_theme_preprocess_oe_corporate_blocks_ec_footer(array &$variables): v
       ],
     ];
   }
+
+  $variables['custom_footer'] = _oe_theme_preprocess_oe_corporate_blocks_site_specific_footer($variables);
+}
+
+/**
+ * Implements hook_preprocess_oe_corporate_blocks_eu_footer().
+ */
+function oe_theme_preprocess_oe_corporate_blocks_eu_footer(array &$variables): void {
+  $variables['custom_footer'] = _oe_theme_preprocess_oe_corporate_blocks_site_specific_footer($variables);
+}
+
+/**
+ * Prepare data for site specific footer preprocess.
+ */
+function _oe_theme_preprocess_oe_corporate_blocks_site_specific_footer(array $variables): array {
+  // Site specific footer.
+  $site_specific_footer = $variables['site_specific_footer'];
+  if (empty($site_specific_footer)) {
+    return [];
+  }
+  if (!empty($site_specific_footer['site_identity'])) {
+    $identity['title'] = $site_specific_footer['site_identity'];
+  }
+  if (!empty($site_specific_footer['social_links'])) {
+    $identity['follow'] = ['label' => t('Follow us:')];
```

Comment:
We need to keep track of strings that need translations here. Do we already have a ticket for this? Probably we do right?

Reply:
Thank you for highlighting this string. I've added this in OPENEUROPA-2427 ticket.